code/modules/mob/living/carbon/alien/humanoid/caste/hunter.dm

```diff
@@ -11,7 +11,7 @@
 	..()
 
 /mob/living/carbon/alien/humanoid/hunter/movement_delay()
-	. = -1		//hunters are sanic
+	. = -0.3	//hunters are sanic

```

Comment:
REEEEEEE

Reply:
some things got moved over incorrectly from mrperson's pr, or this was in the original for some reason